### PR TITLE
Only `push` new `tag` on non-markdown changes

### PR DESCRIPTION
**Pull request by paddyroddy**
I'm not 100% sure if this will work, but let's give it a go. Again, this should reduce some tags. For example, UCL-MIRSG/.github#43 doesn't really need to bump tag.

### CODE REVIEW

File: .github/workflows/tags.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - main
+    paths:
+      - '**/*'
+      - '!**/*.md'
```
Comment (paddyroddy) [inline .github/workflows/tags.yml:9]: This reads all files, not markdown - i.e. if only markdown it shouldn't run

**Review by p-j-smith — APPROVED** [review]
hopefully this will reduce the noise a bit!

Could we also ignore the `.github` folder?

**Comment by paddyroddy** [comment]
> Could we also ignore the `.github` folder?

Good idea. Not sure if this works, but will change and give it a go

**Comment by paddyroddy** [comment]
Well this was a change to something in `.github` and no tag got made. Now to test `*.md` and an actual change.

**Comment by p-j-smith** [comment]
> Good idea. Not sure if this works, but will change and give it a go

I think it worked! It looks like there's no new tag after this pr (which only changed files in `.github/`)

**Pull request merged**